eldritch/test_events.py

```diff
@@ -262,6 +296,151 @@ def testMoveMultipleSpaces(self):
     self.assertEqual(self.char.place.name, "Graveyard")
     self.assertEqual(self.char.movement_points, 1)
 
+  def testIllegalMoveMultipleSpaces(self):
+    self.char.movement_points = 1
+    movement = Sequence(
+        [MoveOne(self.char, dest) for dest in ["Easttown", "Rivertown", "Graveyard"]],
+        self.char,
+    )
+    self.assertFalse(movement.is_resolved())
+    self.assertEqual(self.char.movement_points, 1)
+
+    self.state.event_stack.append(movement)
+    self.resolve_until_done()
+
+    self.assertTrue(movement.is_resolved())
+    self.assertEqual(self.char.place.name, "Easttown")
+    self.assertEqual(self.char.movement_points, 0)
```

Comment:
Since you added the `moved` attribute in this PR, maybe also assert that the `moved` attribute gets set correctly?

Reply:
Would still like for the `moved` attribute to be tested.